build.psm1

```diff
@@ -2214,16 +2214,54 @@ function Start-CrossGen {
             # Generate the ngen assembly
```

Comment:
@TravisEz13, your last commit had 3 failures in `PowerShell-CI-macos`
Test-Connection.Ping.Force IPv4 with implicit PingOptions
```powershell
Expected exactly '10.79.8.150', but got 10.79.1.193.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 83
83:             $result[0].Address | Should -BeExactly $realAddress
```
Test-Connection.MTUSizeDetect.MTUSizeDetect works
```powershell
Value cannot be null. (Parameter 'replyResult')
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 234
234:             $result = Test-Connection $hostName -MtuSize

```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 255
255:             $result[0] | Should -BeOfType Microsoft.PowerShell.Commands.TestConnectionCommand+TraceStatus

```